x-pack/plugins/security_solution/scripts/run_cypress/parallel_serverless.ts

```diff
@@ -325,9 +332,30 @@ function waitForKibanaLogin(kbUrl: string, credentials: Credentials): Promise<vo
   return pRetry(fetchLoginStatusAttempt, retryOptions);
 }
 
+const getOverridedProductTypes = (
```

Comment:
This name has a typo, it should be `getOverriddenProductTypes`. But it would also be okay to call it `getProductTypes`.